### PR TITLE
chore(schematics): migrate axes

### PR DESCRIPTION
**Pull request by AndreiBelokopytov**
Part of https://github.com/taiga-family/taiga-ui/issues/5808

### CODE REVIEW

File: projects/cdk/schematics/ng-update/v4/steps/templates/migrate-axes.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    elements.forEach(({attrs, sourceCodeLocation}) => {
+        const axesNewAttr = attrs.find((attr) => attr.name === AXES_NEW_ATTRIBUTE);
+
+        if (!sourceCodeLocation) {
+            return;
+        }
+
+        if (axesNewAttr) {
+            removeAttrs([axesNewAttr], sourceCodeLocation, recorder, templateOffset);
```
Comment (nsbarsukov) [inline projects/cdk/schematics/ng-update/v4/steps/templates/migrate-axes.ts:37]: Let's just extend `withTagNames` array of this object:
https://github.com/taiga-family/taiga-ui/blob/78ef961d677fd8056599af2df03cbd1abb33c1e5/projects/cdk/schematics/ng-update/v4/steps/constants/attrs-to-replace.ts#L234-L240

with the following object

```ts
{
    from: {
        attrName: 'new',
        withTagNames: ['tui-progress-circle', 'tui-axes'],
    },
    to: {attrName: ''},
},
```

**Review by vladimirpotekhin — APPROVED** [review]

**Review by splincode — APPROVED** [review]

File: projects/cdk/schematics/ng-update/v4/steps/templates/migrate-axes.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        } else {
+            recorder.insertLeft(
+                sourceCodeLocation.startOffset + templateOffset,
+                '<!-- TODO: (Taiga UI migration) labels positioning on x axes were updated. Add empty string to axisXLabels array. See https://taiga-ui.dev/charts/axes -->\n',
```
Comment (nsbarsukov) [inline projects/cdk/schematics/ng-update/v4/steps/templates/migrate-axes.ts:41]: Let's extend this array 
https://github.com/taiga-family/taiga-ui/blob/78ef961d677fd8056599af2df03cbd1abb33c1e5/projects/cdk/schematics/ng-update/v4/steps/constants/html-comments.ts#L3

with the following object
```ts
{
    tag: 'tui-axes',
    filterFn: (el) => !hasElementAttribute(el, 'new'),
    comment:
        'labels positioning on x axes were updated. Add empty string to axisXLabels array. See https://taiga-ui.dev/charts/axes',
},
```

`filterFn` is not supported for this constant.
But feel free to add this feature.

**Comment by nsbarsukov** [comment]
Okay, not today)))

File: projects/cdk/schematics/ng-update/v4/steps/templates/migrate-axes.ts (hunk 2 above)
Comment (AndreiBelokopytov) [reply projects/cdk/schematics/ng-update/v4/steps/templates/migrate-axes.ts:41]: Unfortunately, I missed your comment and PR is already closed

**Pull request merged**